### PR TITLE
Sb 4854 all channel filters

### PR DESCRIPTION
**Pull request by loganathan1989**

### CODE REVIEW

File: src/gulpfile.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ var jasmineNode = require('gulp-jasmine-node')
 var istanbul = require('gulp-istanbul')
```
Comment (rajeevsathish) [inline src/gulpfile.js]: please revert this file change

File: src/gulpfile.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ var jasmineNode = require('gulp-jasmine-node')
...
 
 var paths = {
-  scripts: ['middlewares/*.js', 'models/*.js', 'routes/*.js', 'service/*.js', 'app.js', '!service/conceptService.js'],
+  scripts: ['middlewares/*.js', 'helpers/*.js', 'models/*.js', 'routes/*.js',
```
Comment (rajeevsathish) [inline src/gulpfile.js]: please revert this file changes

File: src/test/helpers/configHelperSpec.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+var configHelper = require('../../helpers/configHelper')
```
Comment (rajeevsathish) [inline src/test/helpers/configHelperSpec.js]: once refactored can you please change the Spec

**Comment by loganathan1989** [comment]
refactored with promises

**Pull request merged**